soc/xtensa/esp32/soc.c

```diff
@@ -36,28 +58,29 @@ void __attribute__((section(".iram1"))) __start(void)
 	__asm__ __volatile__ (
 		"wsr %0, vecbase"
 		:
-		: "r"(&_init_start));
+		: "r" (&_init_start));
```

Comment:
Those whitespace changes do not affect "reboot" functionality, so they should not be part of the commit.

Reply:
We have fixed a bunch of code style in another commit. This diff will then be bypassed.